editor/core/state/index.ts

```diff
@@ -53,6 +54,8 @@ const _state: State = {
   dancerNames: [],
   partTypeMap: {},
   colorMap: {},
+
+  partGroups: {},
```

Comment:
I think `partGroups` should not be put in global state.

Reply:
done